docs/src/@primer/gatsby-theme-doctocat/components/example.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+function Example({src}) {
+  return <div className="Box p-3" dangerouslySetInnerHTML={{ __html: src}}></div>
```

Comment:
Here I had to use `dangerouslySetInnerHTML` instead of `{children}` because the theme was somehow messing with our example tags.

E.g:
`<details-menu class="x">` became `<details-menu classname="x">` and `<u>` became `<u class="List-some-md5">`

Even though this is _dangerous_ we control what is being set as the inner HTML, so it is fine to use it

maybe @colebemis has an idea on how to fix this issue another way

Reply:
`dangerouslySetHTML` seems fine to me 👍